feature-libs/quote/components/summary/actions/quote-summary-actions.component.ts

```diff
@@ -296,6 +320,16 @@ export class QuoteSummaryActionsComponent
     this.quoteFacade.requote(quoteId);
   }
 
+  /**
+   * Determines if there is any document attached with the quote.
+   *
+   * @param attachments - an array of attachments to the quote
+   * @returns - if the document is present, returns 'true', otherwise 'false'.
+   */
+  isAttachmentPresent(attachments: QuoteAttachment[]): boolean {
+    return attachments.length > 0;
```

Comment:
We cannot assume that attachments are always present (dumps if this is not the case)
```suggestion
    return attachments?.length > 0;
```